REFERENCE.md

```diff
@@ -0,0 +1,195 @@
+            "user_id":      "$ANONYMIZED_USER_ID"
+        },
+
+        // secret
+        "74c5fd430cf1242a527f6223aebd42d30464be22",
+
+        // request details
+        {
+            "type":       "local_practice",
+            "state":      "initial",
+            "questions":  [
+                {
+                    "response_id":         "60005",
+                    "type":                "association",
+                    "stimulus":            "Match the cities to the parent nation.",
+                    "stimulus_list":       ["London", "Dublin", "Paris", "Sydney"],
+                    "possible_responses":  ["Australia", "France", "Ireland", "England"],
+                    "validation": {
+                    "score": 1,
+                        "value": ["England", "Ireland", "France", "Australia"]
+                    }
+                }
+            ]
+        }
+    );
+
+    res.render("index", { request: request } );
+});
+
+app.listen(3000, function () {
+    console.log('Example app listening on port 3000!');
+});
+```
+
+``` html
+index.ejs:
+
+// Pass the object to the initialisation of any Learnosity API, in this example the Questions API
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Learnosity SDK - NodeJS</title>
+</head>
+<body>
+<span class="learnosity-response question-60005"></span>
+<script src="//questions.learnosity.com/?v2"></script>
+<script>
+	var request = <%- JSON.stringify(request) %>
+	console.log(request);
+    var questionsApp = LearnosityApp.init(request);
+</script>
+</body>
+</html>
+```
+
+``` json
+package.json:
+
+{
+  "name": "nodeapp",
+  "version": "1.0.0",
+  "description": "Test NodeJS sdk",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "ejs": "^2.5.7",
+    "express": "^4.16.2",
+    "learnosity-sdk-nodejs": "git+https://github.com/Learnosity/learnosity-sdk-nodejs.git#v0.5.0"
+  }
+}
+```
+
+```
+run node.js application: node app.js
+check browser: http://localhost:3000/
+```
+
+#### Init() Arguments
+**service**<br>
+A string representing the Learnosity service (API) you want to integrate with. Valid options are:
+
+* assess
+* author
+* data
+* events
+* items
+* questions
+* reports
+
+**security**<br>
+An array^ that includes your *consumer_key* but does not include your *secret*. The SDK sets defaults for you, but valid options are:
+
+* consumer_key (mandatory)
+* domain (mandatory)
+* timestamp (optional - the SDK will generate this for you)
+* user_id (optional - not necessary for all APIs)
+
+^Note – the SDK accepts a JSON string and native JavaScript objects.
+
+**secret**<br>
+Your private key, as provided by Learnosity.
+
+**request**<br>
+An optional associative array^ of data relevant to the API being used. This will be any data minus the security details that you would normally use to initialise an API.
```

Comment:
```suggestion
Optional data^ relevant to the API being used. This will be any data minus the security details that you would normally use to initialise an API.
```

Reply:
Fixed here: https://github.com/Learnosity/learnosity-sdk-nodejs/pull/8/commits/fbc4ffefb773eadc90cd39721279ae8d99a05f2d